api/toko-rest-api.yaml

```diff
@@ -1,52 +1,85 @@
----
-swagger: "2.0"
+openapi: 3.0.1
 info:
-  description: "Toko REST API methods description"
-  version: "0.0.1"
-  title: "Toko REST API"
+  title: Toko REST API
+  description: Toko REST API methods description
   contact:
-    email: "dauren.del@gmail.com"
+    email: dauren.del@gmail.com
   license:
-    name: "Apache 2.0"
-    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
-host: "api.toko.kz"
-basePath: "/v1"
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.0.1
+servers:
+  - url: //api.toko.kz/v1
 tags:
-  - name: "products"
-    description: "Product API to manage user's product catalog"
-  - name: "Users"
-    description: "Operations about user"
-
+  - name: products
+    description: Product API to manage user's product catalog
+  - name: Users
+    description: Operations about user
 paths:
-  # Products API
-  "/products":
+  /products:
     get:
       tags:
-        - "products"
-      summary: "Retrieve all products"
-      operationId: "getProducts"
-      produces:
-        - "application/json"
+        - products
+      summary: Retrieve all products
+      operationId: getProducts
       responses:
-        "200":
+        200:
           description: OK
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/Product'
-
-definitions:
-  Product:
-     type: object
-     properties:
-       id:
-         type: integer
-         format: int64
-       imageLink:
-         type: string
-         format: uri
-       name:
-         type: string
-       price:
-         type: number
-         format: double
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Product'
+  /user:
```

Comment:
Why did you use a singular noun for path? In [this article](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-use-nouns-instead-of-verbs-in-endpoint-paths), they use plural noun for example with articles